### PR TITLE
Try to fix protocol wiki link

### PR DESCRIPTION
**Pull request by powerjg**

### CODE REVIEW

**Comment by powerjg** [comment]
@ayaz91, Not sure if this worked as I can't seem to get this branch to show up on readthedocs. Do you know how to do this?

Also, there are a ton of broken links in the files I edited on the plane. Could you go through and fix them?

File: docs/artifacts.md [hunk 1]
```diff
@@ -134,3 +135,31 @@ You can use getByName() method to search database for artifacts using the name a
 import gem5art.artifact
 for i in gem5art.artifact.getByName("gem5"):print(i)
 ```
+
+## Artifacts API Documentation
+```eval_rst
+Artifact
```
Comment (powerjg) [inline docs/artifacts.md]: Can you make this more descriptive? Maybe Artifact Module?

File: docs/index.md [hunk 2]
```diff
@@ -1,41 +1,74 @@
+```eval_rst
```
Comment (powerjg) [inline docs/index.md:1]: This file seems like there might have been a rebasing issue, but it looks like it's right in the end, so NBD.

**Review by powerjg — COMMENTED** [review]
One minor comment.

**Comment by aakahlow** [comment]
@powerjg So, I have updated the source to stick to .md as the primary language to use. So, all .rst files are now removed, though at few places I am using .rst syntax embedded in .md code. Moreover, the broken links are fixed as well.

File: docs/artifacts.md (hunk 1 above)
Comment (aakahlow) [reply docs/artifacts.md]: Ok!

**Comment by powerjg** [comment]
Race condition :).

File: docs/index.md (hunk 2 above)
Comment (powerjg) [reply docs/index.md:1]: Nevermind. I see that this was moved from index.rst

**Comment by aakahlow** [comment]
> @ayaz91, Not sure if this worked as I can't seem to get this branch to show up on readthedocs. Do you know how to do this?
> 
For readthedocs, we need to activate each branch individually that we want to see on RTD server. I have added this branch, and you can look at the build here:

https://gem5art.readthedocs.io/en/link-fix/index.html

There might be a way to activate all branches of a repo, though I am not aware of that at the moment. Also, I am not sure if we want to enable that

**Comment by powerjg** [comment]
Gotcha. You addressed all of my concerns :) Merge whenever you're happy!

**Pull request merged**